scout/adapter/mongo/query.py

```diff
@@ -163,7 +164,15 @@ def build_query(self, case_id, query=None, variant_ids=None, category='snv'):
                 gene_query = self.gene_filter(query, mongo_query)
 
             elif criterion == 'chrom' and query.get('chrom'): # filter by coordinates
-                self.coordinate_filter(query, mongo_query)
+                LOG.debug('Adding genomic coordinates to the query')
+                mongo_query['chromosome'] = query['chrom']
+
+                coordinate_query = None
+                if (query.get('start') and query.get('end')):
+                    if category == 'sv':
+                        coordinate_query = [self.sv_coordinate_filter(query, mongo_query)]
+                    else:
+                        self.coordinate_filter(query, mongo_query)
```

Comment:
Its a little strange that the coordinate_filter and sv_coordinate_filter have different calling mechanics. Either unify or change the name of the sv_coordinate_filter to sth like sv_coordinate_query.

Reply:
I'd rather not unify them because they work in different ways, I'll rename the sv one to sv_coordinate_query

Reply:
✅